pkg/glusterblock/identityserver.go

```diff
@@ -0,0 +1,45 @@
+package glusterblock
+
+import (
+	"context"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/golang/glog"
+)
+
+// IdentityServer struct of Glusterfs CSI driver with supported methods of CSI identity server spec.
+type IdentityServer struct {
```

Comment:
if we do some refractor this can also be the common file between  both (this can be done as  cleanup if  you want)

Reply:
I will do code refactor in separate patch